Remotion/Web/UnitTests/Core/UI/Controls/DropDownMenuImplementation/Rendering/DropDownMenuRendererTest.cs

```diff
@@ -31,6 +31,7 @@
 using Remotion.Web.UI.Controls.DropDownMenuImplementation;
 using Remotion.Web.UI.Controls.DropDownMenuImplementation.Rendering;
 using Remotion.Web.UI.Controls.Rendering;
+using ButtonType = Remotion.Web.UI.Controls.ButtonType;
```

Comment:
Why do need this in the Unittests? Didn't do a checkout, so it just seems strange?

Reply:
Because there is a `System.Web.UI.WebControls.ButtonType`